x-pack/plugins/cloud_integrations/cloud_experiments/public/plugin.ts

```diff
@@ -116,14 +135,13 @@ export class CloudExperimentsPlugin
     if (this.flagOverrides && has(this.flagOverrides, configKey)) {
       return get(this.flagOverrides, configKey, defaultValue) as Data;
     }
-    if (!this.launchDarklyClient) return defaultValue; // Skip any action if no LD User is defined
-    await this.launchDarklyClient.waitForInitialization();
-    return this.launchDarklyClient.variation(configKey, defaultValue);
+    if (!this.launchDarklyClient) return defaultValue; // Skip any action if no LD Client is defined
```

Comment:
NIT/optional: code style consistency

```ts
if (!this.launchDarklyClient) { 
  return defaultValue;
}
```

Reply:
Addressed in commit [elastic/kibana@`48eaafc` (#143002)](https://github.com/elastic/kibana/pull/143002/commits/48eaafc61032a624a22f6062b56833cca1afa361)